packages/rest-explorer/src/rest-explorer.component.ts

```diff
@@ -28,6 +28,13 @@ export class RestExplorerComponent implements Component {
 
     this.registerControllerRoute('get', explorerPath, 'indexRedirect');
     this.registerControllerRoute('get', explorerPath + '/', 'index');
+    if (restExplorerConfig.relativeOpenApiSpecEndpoint !== false) {
```

Comment:
I have mixed feelings about the option name `relativeOpenApiSpecEndpoint`, I feel it's describing implementation details instead of the intent. How about `useSelfHostedSpec` or `selfHostSpec`?

Reply:
Not sure why I didn't think of this before, `useSelfHostedSpec` is _much_ clearer & simpler :grinning: